tests/fixtures/rules/mustache-spacing/invalid/always-after-expression/snippet-render01-output.svelte

```diff
@@ -0,0 +1,4 @@
+<!-- prettier-ignore -->
+{#snippet foo() }
```

Comment:
I don't think we don't need to add space before closing bracket.

```suggestion
{#snippet foo()}
```

The docs also doesn't have a space and I think it's more natural for me.

![image](https://github.com/sveltejs/eslint-plugin-svelte/assets/19153718/cc0d3f9d-6417-4480-b937-073b57e43d52)

https://svelte-5-preview.vercel.app/docs/snippets